charmander/routes/pizza/crud.py

```diff
@@ -1,5 +1,5 @@
 """
-Example CRUD routes.
+Pizza CRUD routes.
```

Comment:
Behind the scenes the `crud` module here takes care of the `flask` internals to make routes out of the controller functions mentioned here.